src/java.base/share/classes/javax/security/auth/x500/X500Principal.java

```diff
@@ -91,7 +91,7 @@ public final class X500Principal implements Principal, java.io.Serializable {
      *
      * NOTE: The constructor is package private. It is intended to be accessed
      * using privileged reflection from classes in sun.security.*.
-     * Currently referenced from sun.security.x509.X500Name.asX500Principal().
+     * Currently, referenced from sun.security.x509.X500Name.asX500Principal().
```

Comment:
Change to "Currently, it is referenced from ..."

Reply:
Okay.